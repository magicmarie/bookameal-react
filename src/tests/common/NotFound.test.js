import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../components/common/NotFound";

describe("NotFound component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });
  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
