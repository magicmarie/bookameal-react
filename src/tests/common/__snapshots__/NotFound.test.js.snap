// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          404: Page Not Found
</h1>,
        <Link
          href="/"
          replace={false}
          to="/"
>
          Home
</Link>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404: Page Not Found",
        },
        "ref": null,
        "rendered": "404: Page Not Found",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Home",
          "href": "/",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "Home",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            404: Page Not Found
</h1>,
          <Link
            href="/"
            replace={false}
            to="/"
>
            Home
</Link>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404: Page Not Found",
          },
          "ref": null,
          "rendered": "404: Page Not Found",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Home",
            "href": "/",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "Home",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
